src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -64,7 +64,7 @@
      * @param cramFile CRAM file to open
      * @param inputStream   CRAM stream to read
      */
-    public CRAMFileReader(final File cramFile, final InputStream inputStream) {
+    public CRAMFileReader(final File cramFile, final InputStream inputStream) throws FileNotFoundException, IOException {
```

Comment:
it's bad style to have 2 exceptions declared and one of them be a subclass of the other: `FileNotFoundException extends IOException`